### ISSUE TITLE
Make icon metadata available via a package

### ISSUE THREAD

**Issue by micahgodbolt** [microsoft/fluentui-system-icons#788]
I'm looking to use the icons meta data while rendering our docs, specifically to allow users to search by metaphor to find the icon. It doesn't make sense for each package to separately publish this, so is this something that could be created as its own package?

**Comment by willchavez** [comment]
Hey @micahgodbolt can you expand on this? Would a separate package with all the metadata compressed into a single file fit your need? Or separate json files fine?

**Comment by micahgodbolt** [comment]
yeah, a single file would be fine. I'd love to be able to take an icon like AccessibilityErrorFilled, and look up all of the related terms. This would greatly improve search, as we can only currently search by exact name.

https://react.fluentui.dev/?path=/docs/icons-catalog--docs

![Image](https://github.com/user-attachments/assets/0af1284d-b2f5-45f9-85c2-4b4124859816)

**Comment by MannGo1** [comment]
Hi Fluent Team. we also have a requirement where we give user ability to assign an icon corresponding to a property displayed on a card. For this user needs to search the icons. We were thinking to use the metaphors associated each icon from the icon's metadata.json file. But since these metaphors are not accessible programatically we would have to download the metadata.json and use that for each icon. So we would also like to have these feature, so that we can access these metaphors using a pacakge.
